FWCore/Framework/interface/OccurrenceTraits.h

```diff
@@ -423,6 +429,7 @@ namespace edm {
   class OccurrenceTraits<ProcessBlockPrincipal, BranchActionProcessBlockInput> {
   public:
     using MyPrincipal = ProcessBlockPrincipal;
+    using TransitionInfoType = ProcessBlockTransitionInfo;
     using Context = GlobalContext;
     static bool constexpr isEvent_ = false;
     static Transition constexpr transition_ = Transition::BeginProcessBlock;
```

Comment:
We don't have a separate ProcessBlockInput transition?

Reply:
I'm thinking about this. Before I recently merged in the concurrent EventSetup module pull request, there was no reason to add this transition into the Transition enum. It wasn't needed anywhere.  Still thinking, I'm not sure.

Reply:
I added a new value to the edm::Transition enum. Since OccurrenceTraits now wants this value, it makes since to add it.

Reply:
I meant "it makes sense to add it"